<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HighwayTypeWidget</class>
 <widget class="QWidget" name="HighwayTypeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1629</width>
    <height>56</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>2</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="0" column="1">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Priority</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" rowspan="4">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Highway Tag Value</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Maximum Speed</string>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Default City Speed</string>
     </property>
    </widget>
   </item>
   <item row="0" column="6">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Average Speed</string>
     </property>
    </widget>
   </item>
   <item row="0" column="7" rowspan="4">
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="9">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Turning Into</string>
     </property>
    </widget>
   </item>
   <item row="0" column="10" rowspan="4">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="12">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Left Turn</string>
     </property>
    </widget>
   </item>
   <item row="0" column="13">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Straight</string>
     </property>
    </widget>
   </item>
   <item row="0" column="14">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>Right Turn</string>
     </property>
    </widget>
   </item>
   <item row="0" column="15" rowspan="4">
    <widget class="Line" name="line_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="16">
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>Left Turn Penalty</string>
     </property>
    </widget>
   </item>
   <item row="0" column="17">
    <widget class="QLabel" name="label_12">
     <property name="text">
      <string>Right Turn Penalty</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3" rowspan="3">
    <widget class="QLineEdit" name="type">
     <property name="text">
      <string>motorway</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4" rowspan="3">
    <widget class="QSpinBox" name="maxSpeed">
     <property name="suffix">
      <string> km/h</string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
    </widget>
   </item>
   <item row="1" column="5" rowspan="3">
    <widget class="QSpinBox" name="citySpeed">
     <property name="suffix">
      <string> km/h</string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
    </widget>
   </item>
   <item row="1" column="6" rowspan="3">
    <widget class="QSpinBox" name="averageSpeed">
     <property name="suffix">
      <string> %</string>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>80</number>
     </property>
    </widget>
   </item>
   <item row="1" column="11" rowspan="2">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Look out for Highways with</string>
     </property>
    </widget>
   </item>
   <item row="1" column="12" rowspan="3">
    <widget class="QComboBox" name="otherCarsLeft">
     <item>
      <property name="text">
       <string>-</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Higher Priority</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Equal or Higher Priority</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="13" rowspan="3">
    <widget class="QComboBox" name="otherCarsStraight">
     <item>
      <property name="text">
       <string>-</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Higher Priority</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Equal or Higher Priority</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="14" rowspan="3">
    <widget class="QComboBox" name="otherCarsRight">
     <item>
      <property name="text">
       <string>-</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Higher Priority</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Equal or Higher Priority</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="16" rowspan="3">
    <widget class="QSpinBox" name="leftPenalty">
     <property name="suffix">
      <string> s</string>
     </property>
    </widget>
   </item>
   <item row="1" column="17" rowspan="3">
    <widget class="QSpinBox" name="rightPenalty">
     <property name="suffix">
      <string> s</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QToolButton" name="close">
     <property name="text">
      <string>x</string>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="9">
    <widget class="QCheckBox" name="pedestrian">
     <property name="text">
      <string>Pedestrian Penalty</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSpinBox" name="priority">
     <property name="maximum">
      <number>10000</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>close</sender>
   <signal>clicked()</signal>
   <receiver>HighwayTypeWidget</receiver>
   <slot>deleteLater()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1143</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>1136</x>
     <y>71</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
